site/sass/components/_nav.phosphor.scss

```diff
@@ -0,0 +1,37 @@
+// Navigation Styling
+// This is pretty complex as we have two almost entirely separate sets of styling represented here
+
+.nav {
+  &__link {
+    @include minLangMQ($bkpt-map--header--large) {
+      &:hover,
+      &:focus {
+        box-shadow: none;
+      }
+
+      &:focus {
+        box-shadow: none;
+      }
+
+      &[aria-expanded='true'] {
+        &::after {
+          border-color: transparent transparent var(--blue-850);
+        }
+
+        &::before {
+          border-color: transparent transparent var(--blue-850);
+        }
+      }
+    }
+  }
+
+  &__secondary {
+    border: solid 2px var(--blue-850);
+    // margin: 0 20px;
+
+    @include minLangMQ($bkpt-map--header--large) {
+      margin: 0 10px;
+      width: calc(100vw - 40px);
+    }
```

Comment:
Non-color items changing

Reply:
The nav specifically had inset/margin added in design for added impact (1/2)

Reply:
I actually tweaked this to be more of a color swap and removed the margins. this resolved mobile issues that were missed.